core/src/main/java/dev/streamx/cli/license/LicenseConfig.java

```diff
@@ -0,0 +1,41 @@
+package dev.streamx.cli.license;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import dev.streamx.cli.license.input.AcceptingStrategy;
+import dev.streamx.cli.license.input.FixedValueStrategy;
+import dev.streamx.cli.license.input.StdInLineReadStrategy;
+import io.quarkus.arc.DefaultBean;
+import io.quarkus.arc.profile.IfBuildProfile;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.Dependent;
+import jakarta.enterprise.inject.Produces;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Dependent
+class LicenseConfig {
+
+  @ApplicationScoped
+  @LicenseProcessing
+  ObjectMapper licensesProcessingObjectMapper() {
+    ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+
+    return objectMapper;
+  }
+
+  @Produces
+  @IfBuildProfile("test")
+  AcceptingStrategy fixedValue(
```

Comment:
why do we need this?

Reply:
For test purposes.

I think Testing I/O should be excluded from tests and tests should base on abstraction, which is at least configurable.